lib/muledump.js

```diff
@@ -1,6 +1,593 @@
 (function($, window) {
 
+<<<<<<< HEAD
```

Comment:
Looks like you didn't merge correctly... You might find it easier to update your master to match atomizer's, then create a new branch and recreate your changes there.